src/main/java/de/zalando/aruha/nakadi/repository/db/EventTypeCachedRepository.java

```diff
@@ -0,0 +1,49 @@
+package de.zalando.aruha.nakadi.repository.db;
+
+import de.zalando.aruha.nakadi.domain.EventType;
+import de.zalando.aruha.nakadi.exceptions.NakadiException;
+import de.zalando.aruha.nakadi.exceptions.NoSuchEventTypeException;
+import de.zalando.aruha.nakadi.repository.EventTypeRepository;
+
+import java.util.List;
+
+public class EventTypeCachedRepository implements EventTypeRepository {
+
+    private final EventTypeRepository repository;
+
+    private final EventTypeCache cache;
+
+    public EventTypeCachedRepository(EventTypeRepository repository,
```

Comment:
`CachingEventTypeRepository` or `CachedEventTypeRepository` since it implements `EventTypeRepository`?